docs/articles/expensify-classic/manage-employees-and-report-approvals/Approval-Workflows.md

```diff
@@ -106,4 +106,4 @@ Category and Tag approvers are inserted at the beginning of the approval workflo
 
 
 ### Workflow enforcement
-- If you want to ensure your employees cannot override the workflow you set - enable workflow enforcement by following the steps below. As a Workspace Admin, you can choose to enforce your approval workflow by going.
\ No newline at end of file
+- If you want to ensure your employees cannot override the workflow you set - enable workflow enforcement by following the steps below. As a Workspace Admin, you can choose to enforce your approval workflow by going.
```

Comment:
This seems like an incomplete thought 😅

Reply:
hehe true, I'm not even sure why this is showing as diff if the lines are the same